src/main/java/seedu/cardli/commands/deck/EditCardCommand.java

```diff
@@ -58,6 +65,16 @@ public static String prepareCardIndex(String card, Deck deck) throws CardLiExcep
         return card;
     }
 
+    /**
+     * Returns the checked arguments for EditCardCommand.
```

Comment:
Maybe it's better to say "Returns the arguments for EditCardCommand if accepted".

Reply:
thanks it does sound better